site/docs/contribute/logging.md

```diff
@@ -33,6 +33,30 @@ Log levels should be used as follows:
 - debug (zerolog.DebugLevel, 0) - Additional information needed to help identify problems (typically used during testing)
 - trace (zerolog.TraceLevel, -1) - For tracing step by step flow of control (typically used during development)
 
+## JSON Logging Standard Format
+All Fybrik components should generate logging information in a standard format. This information will be used by different actors for different purposes, so as much relevant information as possible needs to be captured in a consistent format.
+
+We list all mandatory and optional fields to be used by all Fybrik components. In addition to the fields we list, Fybrik components may include extra fields as needed.
+
+### Mandatory Fields
+- level - log level (‘panic’, ‘fatal’, ‘error’, ‘warn’, ‘info’, ‘debug’, or ‘trace’)
+- time - timestamp  of the log event.  Timestamps  should  be in ISO8601  format with  time offset from  UTC or timezone. Example: ‘2022-02-16T10:46:21+02:00’
+- caller - the code line which generated the error (file name + line number). Example: manager/main.go:319
+
+### Optional Fields
+- app.fybrik.io/app-uuid - unique identifier for kubernetes FybrikApplication, used to correlate log messages across components
```

Comment:
used to correlate log messages across components **for a particular FybrikApplication instance**.  It is also unique over time so one may differentiate between FybrikApplication instances with the same name created at different times.